engines/bastion_katello/app/assets/javascripts/bastion_katello/check-current-organization.run.js

```diff
@@ -0,0 +1,53 @@
+/**
+ * Copyright 2014 Red Hat, Inc.
+ *
+ * This software is licensed to you under the GNU General Public
+ * License as published by the Free Software Foundation; either version
+ * 2 of the License (GPLv2) or (at your option) any later version.
+ * There is NO WARRANTY for this software, express or implied,
+ * including the implied warranties of MERCHANTABILITY,
+ * NON-INFRINGEMENT, or FITNESS FOR A PARTICULAR PURPOSE. You should
+ * have received a copy of GPLv2 along with this software; if not, see
+ * http://www.gnu.org/licenses/old-licenses/gpl-2.0.txt.
+ **/
+
+
+(function () {
+
+    /**
+     * @ngdoc run
+     * @name Bastion.run:CheckCurrentOrganization
+     *
+     * @description
+     *   Handles checking if a given feature is enabled. Provides this functionality on
+     *   the root scope and also checks routes to send the user to a 403.
+     */
+    function CheckCurrentOrganization($rootScope, $window, CurrentOrganization) {
+        var fencedPages = [
+            'products',
+            'activation-keys',
+            'environments',
+            'subscriptions',
+            'gpg-keys',
+            'sync-plans',
+            'content-views',
+            'errata',
+            'content-hosts',
+            'host-collections'
```

Comment:
We'll have to add support to bastion to allow adding constants via HTML and, if, in the future we remove the need to have an organization selected (supporting the any context case) we will have to split that into two pieces since we'd need to be able to remove the need to check organization from pages.